### PR TITLE
feat: Add shell cmd for switch between LTE-M and NB-Iot mode

### PR DESCRIPTION
**Pull request by boundlesscalm**
Signed-off-by: Pascal Pickel <Pascal.Pickel@nordicsemi.no>

### CODE REVIEW

File: src/nat_cmd.c [hunk 1]
```diff
@@ -170,14 +170,43 @@ static void handle_stop_test(const struct shell *shell, size_t argc,
 	shell_print(shell, "Test stopped\n");
 }
 
+static void handle_get_network_mode(const struct shell *shell, size_t argc,
+				    char **argv)
+{
+	shell_print(shell, "Network mode: %d\n", get_network_mode());
```
Comment (coderbyheart) [inline src/nat_cmd.c]: It would be nice to print either `LTE` or `NB-IoT` instead of an integer.

File: src/nat_cmd.c [hunk 2]
```diff
@@ -170,14 +170,43 @@ static void handle_stop_test(const struct shell *shell, size_t argc,
...
+	long value = strtol(argv[1], NULL, 10);
+	int err = set_network_mode(value);
+
+	if (err == -INVALID_MODE) {
+		shell_print(shell, "Invalid mode\n");
+	} else if (err == -TEST_RUNNING) {
+		shell_print(shell, "Active test - Unable to change mode\n");
+	}
+
+	shell_print(shell, "Changed network mode to %d\n", value);
```
Comment (coderbyheart) [inline src/nat_cmd.c]: Same here, print the identifier of the mode, not just the integer.

**Review by coderbyheart — APPROVED** [review]

File: src/nat_cmd.c (hunk 1 above)
Comment (boundlesscalm) [reply src/nat_cmd.c]: Fixed

File: src/nat_cmd.c (hunk 2 above)
Comment (boundlesscalm) [reply src/nat_cmd.c]: Fixed

**Review by jhn-nordic — APPROVED** [review]

**Pull request merged**